### ISSUE TITLE
Values in Testing alert rule doesn't correspond to Graph value

### ISSUE THREAD

**Issue by IMG-PRCSNG** [grafana/grafana#14979]
### What Grafana version are you using?
Grafana v5.5.0-pre1 (d843044)

### What datasource are you using?
Elasticsearch

### What OS are you running grafana on?
Linux Ubuntu 16.04

### What did you do?
I wrote a query to get 95 percentile of response time and tried to set an alert on it.

### What was the expected result?
While testing the alert rule using a **max()** operator on the query, I expected the values in the response array and the corresponding value on the graph to match.

### What happened instead?
The values don't match for the corresponding timestamp. As seen on the screenshot (Look at highlighted array value and the cursor on graph)

Timestamp 1548138417 = 11:56:57 IST
<img width="1205" alt="screenshot 2019-01-22 at 12 04 04 pm" src="https://user-images.githubusercontent.com/3027352/51517072-97cbce80-1e3f-11e9-8cd0-39984b7c6b2a.png">

My Elasticsearch Query is 
`Metric: Percentiles, responseTime, percents:95`
`Group By: Date Histogram, @timestamp, interval: 1s`

My alert rule is `WHEN max() of query(A, 6m, now - 5m) IS ABOVE 3000`

I am suspecting that the Date Histogram interval is changed to auto when applying the max operator on the query during alert evaluation. I kept it at 1s for plotting the graph for the last 5 minutes.

The values match (test alert response dump and graph) when I increase the alert lookback duration to 5 minutes (query(A, 10m, now - 5m)) and it doesn't match if i keep it as 1 minute (query(A, 6m, now-5m)). Any pointers on how to confirm if this is the issue?

**Comment by marefr** [comment]
Based on your information your looking at the timerange between ~11:52:30 and ~11:53:30 in your alert rule compared with your graph. Closing this, but will consider reopen if I misunderstood and/or you can provide more information that shows this is an actual bug.

**Comment by IMG-PRCSNG** [comment]
I will run the test again and show you the time range. That graph was timeshifted by 5m.

**Comment by IMG-PRCSNG** [comment]
I had highlighted the absolute timestamp in both graph and the alert data just so that we don't have any confusion related to time.

Having said that, I might have jumped to conclusions here by testing only the first value in all my tries. In the tests that I conducted, the second value onwards matched. This might be due to edge effects of the series. I will update this issue with my findings if I find any discrepancies. Thanks for having a look at it!
